events-generator/src/main/java/org/moditect/jfrunit/generator/Generator.java

```diff
@@ -0,0 +1,11 @@
+package org.moditect.jfrunit.generator;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.SOURCE)
+public @interface Generator {
```

Comment:
Do we need that in fact? You can have an AP react to "*", which should be enough to trigger it for the project. You'd just have to track that it was started and ran once, so you don't do it for each annotation it encounters.